### PR TITLE
EMO-6941 fix security vulnerabilities

### PR DESCRIPTION
**Pull request by mike-unitskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Rison Parser and Generator for Jackson
 ======================================
 
-A plugin for the [Jackson streaming JSON processor v2.9.x] (http://wiki.fasterxml.com/JacksonHome) that adds
```
Comment (bdevore17) [inline README.md]: Changes like these don't really belong in this patch. Please remove it and move it to a different PR if we still need it.

File: pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>com.bazaarvoice.commons</groupId>
         <artifactId>bv-opensource-super-pom</artifactId>
-        <version>1.0</version>
+        <version>1.5</version>
```
Comment (bdevore17) [inline pom.xml]: Is this change relevant to the jackson upgrade? Or separate? If they are separate we should move it to separate pull requests.

File: pom.xml [hunk 3]
```diff
@@ -60,7 +60,7 @@
         <dependency>
             <groupId>org.testng</groupId>
             <artifactId>testng</artifactId>
-            <version>6.3.1</version>
+            <version>6.14.3</version>
```
Comment (bdevore17) [inline pom.xml]: Is this change relevant to the jackson upgrade? Or separate? If they are separate we should move it to separate pull requests.

File: pom.xml [hunk 4]
```diff
@@ -71,7 +71,6 @@
                 <!-- Make sure we always turn all warnings on during compilation. -->
                 <plugin>
                     <artifactId>maven-compiler-plugin</artifactId>
-                    <version>2.3.2</version>
```
Comment (bdevore17) [inline pom.xml]: Is this change relevant to the jackson upgrade? Or separate? If they are separate we should move it to separate pull requests.

File: README.md (hunk 1 above)
Comment (mike-unitskyi) [reply README.md]: it just fixes broken links

Comment (bdevore17) [reply README.md]: I would rather it be in a separate PR, just as a matter process. PR's are nearly free, and we can roll all changes into a single release.

Comment (mike-unitskyi) [reply README.md]: ok, will revert that

File: pom.xml (hunk 2 above)
Comment (mike-unitskyi) [reply pom.xml]: will revert that

File: pom.xml (hunk 3 above)
Comment (mike-unitskyi) [reply pom.xml]: will revert

File: pom.xml (hunk 4 above)
Comment (mike-unitskyi) [reply pom.xml]: will revert

**Comment by mike-unitskyi** [comment]
@bdevore17 please check again

**Comment by mike-unitskyi** [comment]
@bdevore17 thanks, will create tickets and PRs on Monday, won't release new version until that
